src/java.base/share/classes/java/net/MulticastSocket.java

```diff
@@ -25,6 +25,10 @@
 
 package java.net;
 
+import org.jspecify.annotations.NullMarked;
```

Comment:
Did you mean to use this on the class?

Reply:
Definitely, thanks. Do you think the rest of the annotations in the class are sufficient to just add it? (That had been my intention, and I'm pushing a commit to do so, but I'll hold off on committing until I hear back.)

Reply:
Yes, the annotations in this file look good to me - including that I don't know what the right answer for the TODOs is.